moc-health-check/moc_health_check.py

```diff
@@ -0,0 +1,313 @@
+#!/usr/bin/env python -u
+import os, sys
+import configparser
+from subprocess import Popen, PIPE
+import test_setup
+import subprocess
+import time
+import logging
+import pfurl
+import json
+import test_setup
+
+
+class Health_Checker:
+    # defining all of required variables
+    def __init__(self):
+        config = configparser.ConfigParser()
+        config.read('moc-health-check/config.cfg')
+        self.RANGE = config.get('ConfigInfo', 'RANGE')
+        self.SIZE = config.get('ConfigInfo', 'SIZE')
+        self.TIMEOUT = config.get('ConfigInfo', 'TIMEOUT')
+        self.THRESHOLD = config.get('ConfigInfo', 'THRESHOLD')
+        self.PATH = os.getcwd()
+        self.WAIT = int(config.get('ConfigInfo', 'WAIT'))
+        self.PMAN_URL = config.get('ConfigInfo', 'PMAN_URL')
+        self.PFIOH_URL = config.get('ConfigInfo', 'PFIOH_URL')
+        self.JID = config.get('ConfigInfo', 'JID')
+        self.success_pfioh_push = 0                               # Success rate of pfioh_push
+        self.success_pman_run = 0                               # Success rate of pman_run
+        self.success_pman_status = 0                               # Success rate of pman_status
+        self.success_pfioh_pull = 0                               # Success rate of pfioh_pull
+
+        # Creating sample files
+        test_setup.check() 
+
+        # Reentrancy testing
+        self.check_job_status()
+
+        """ Log File Reentrancy
+        last_cmd = self.last_line("program.log","root:")
+
+        if(self.reentrancy(last_cmd.strip(), "job_delete")):
+            self.job_delete()
+        """
+
+        # Erasing contents of error log file & program log file
+        self.erase('moc-health-check/error.log', "\n")
+        self.job_delete()
+
+
+    # accessor methods
+    def get_range(self):
+        return self.RANGE
+    
+    def get_threshold(self):
+        return self.THRESHOLD
+
+    # delete a pman job
+    def job_delete(self):
+        data = {   "action": "delete",

```

Comment:
Give an appropriate name for variable `data`. `data` is misleading. `request` sounds more appropriate.